src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java

```diff
@@ -315,6 +318,28 @@ ExitCode parseOptions(List<String> args, ExtendedEventHandler eventHandler) {
     return ExitCode.SUCCESS;
   }
 
+  /**
+   * If --enable_platform_specific_config is true and the corresponding config definition exists,
+   * we should enable the platform specific config.
+   */
+  private boolean shouldEnablePlatformSpecificConfig(
+      OptionValueDescription enablePlatformSpecificConfigDescription,
+      ListMultimap<String, RcChunkOfArgs> commandToRcArgs) {
+    if (enablePlatformSpecificConfigDescription == null ||
+        !(boolean) enablePlatformSpecificConfigDescription.getValue()) {
+      return false;
+    }
+
+    Multiset<String> keys = commandToRcArgs.keys();
```

Comment:
If you inlined this, you wouldn't need to import Multiset.
Also, call `containsKey` instead of `keys().contains` -- it's possible that the latter constructs a set and the former doesn't.

Reply:
Nice, thanks for the suggestion!